react/javascript/demo-extension-sdk/src/__mocks__/render_with_extension.js

```diff
@@ -32,19 +34,25 @@ import { registerHostApi } from '@looker/extension-sdk'
 
 const getExtensionSDK = (extensionSDKOverride) => {
   const extensionSDK = {
+    error: () => {
+      // noop
+    },
+    getContextData: () => {
+      // noop
+    },
+    localStorageGetItem: () => {
+      // noop
+    },
```

Comment:
Did you insert these comments, or did the linter?

Reply:
I did those